packages/core/src/main.js

```diff
@@ -2,4 +2,7 @@ import '@department-of-veterans-affairs/web-components/dist/component-library/co
 
 export * from '@department-of-veterans-affairs/react-components';
 export * from '@department-of-veterans-affairs/web-components';
-export { applyPolyfills } from '@department-of-veterans-affairs/web-components/loader';
+export {
+  applyPolyfills,
+  defineCustomElements,
```

Comment:
Can you elaborate on this change? In https://github.com/department-of-veterans-affairs/component-library/pull/218 we switched from implicitly exporting everything (including `defineCustomElements`) to only explicitly exporting `applyPolyfills` in order to fix a webpack 5 issue in `vets-website`.

![image](https://user-images.githubusercontent.com/2008881/148141116-370fc4e4-0590-44dd-87de-0002b06321fd.png)

If the named export doesn't cause the same issue that the star export did then this is fine.